BioSQL/BioSeqDatabase.py

```diff
@@ -46,7 +46,6 @@ def open_database(driver="MySQLdb", **kwargs):
      - password, passwd - the password to connect with
      - host - the hostname of the database
      - database or db - the name of the database
-
```

Comment:
Please check this is still valid RST. I suspect that we can in fact drop the blank line after "Arguments" provided the bullet point list is indented.